samples/05.custom-components/d.reaction-buttons/index.html

```diff
@@ -116,7 +116,11 @@
         const activityMiddleware = () => next => (...setupArgs) => {
           const [card] = setupArgs;
 
-          if (card.activity.from.role === 'bot') {
+          if (card.activity.type === 'messageReaction') {
+            // Currently, Web Chat do not handle "messageReaction" activity and will show a warning as it is an unknown activity.
```

Comment:
s/do/does

Reply:
Fixed.

The warning was shown in console.

I am happy to see a trend of web devs start looking at the console while working on their page, i.e. now they want zero errors. :)